<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1758</width>
    <height>1254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>912</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Whole Brain Shimming</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_18" stretch="0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16" stretch="0">
             <item>
              <widget class="QGroupBox" name="GREtoB0MapGroup">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>GRE to B₀ Map</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0" colspan="2">
                 <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,2">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                     <widget class="QPushButton" name="LoadB0Map">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                        <horstretch>1</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="text">
                       <string>Load B₀ Map</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="LoadMag">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                        <horstretch>1</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="text">
                       <string>Load Mag</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="SetDicomPath">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                        <horstretch>1</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="text">
                       <string>Set DICOM Path</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="GenerateB0Map">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                        <horstretch>1</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="text">
                       <string>Generate B₀ Map</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="DicomPathLabel">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>2</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="text">
                       <string>DICOM Path</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="DicomPath">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                        <horstretch>4</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>2</horstretch>
               <verstretch>2</verstretch>
              </sizepolicy>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ActiveShimmingGroup">
          <property name="title">
           <string>Active Shimming</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="GenerateCoilProfile">
             <property name="text">
              <string>Generate Coil Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LoadCoilProfile">
             <property name="text">
              <string>Load Coil Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ConfigureObj">
             <property name="text">
              <string>Configure Objective</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LoadMask">
             <property name="text">
              <string>Load Mask</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SimulateOpt">
             <property name="text">
              <string>Simulate Optimization</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ViewMetrix">
             <property name="text">
              <string>View Metrics</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DeployHardware">
             <property name="text">
              <string>Deploy to Hardware</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LoadOptFile">
             <property name="text">
              <string>Load Optimized File</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="TeensyGroup">
          <property name="title">
           <string>Teensy Interface</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QListWidget" name="TeensyInterface">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="OrignalMapsGroup">
          <property name="title">
           <string>Original B₀ Maps</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13"/>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="SimBox">
          <property name="title">
           <string>Simulated B₀ Maps</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14"/>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="OptBox">
          <property name="title">
           <string>Optimized B₀ Maps</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15"/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="Sagittal">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="Coronal">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSlider" name="Axial">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="ColorBarBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Colorbar </string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="MaxLabel">
             <property name="text">
              <string>Max:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="MaxNum">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="MinLabel">
             <property name="text">
              <string>Min:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="MinNumber">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="UnitLabel">
             <property name="text">
              <string>Unit:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="Unit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>ppm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hz</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1758</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
